recipes/amrex/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "amrex" %}
+{% set version = "23.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AMReX-Codes/amrex/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e76e4787735c06ae747eaf57a682b12fed297ef2d5c7727051e20d34e4e79ce3
+  patches:
+    # Install CMake Scripts from Tools/CMake/ to <CMakePkgRoot>/Modules/
+    # https://github.com/AMReX-Codes/amrex/pull/3599
+    - 3599.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ccache        # [unix]
+    - clang         # [win]
+    - cmake
+    - lld           # [win]
+    # TODO: add libxml2 dep in lld-feedstock
+    - libxml2       # [win]
+    - make          # [unix]
+    - ninja         # [win]
+    - libgomp       # [linux]
+    - llvm-openmp   # [osx or win]
```

Comment:
@isuruf since OpenMP has a platform-specific runtime, shouldn't it be in `host:`?